### PR TITLE
 Integrate top story to front page section

### LINKED ISSUE #1964: Integrate Top Story Promo
**Is your feature request related to a problem? Please describe.**
Following https://github.com/bbc/psammead/pull/578 we have a new styling for the 'top story' which can be set by passing a prop

**Describe the solution you'd like**
For the first story, in the first section, pass the `topStory: true` prop to the promo, headline and summary. This should cause the styling to be emphasised <600px

This can likely be done by passing a `section number` and `story number` prop down, using the index key on the 2 `maps` used. Then when they equal `0` and `0`, we know its the first story on the first section :)

**Testing notes**
No cypress test required, visual check only.

Dev insight: Storybook (likely covered already) and unit tests

### PR DESCRIPTION
**Pull request by ghost**
Resolves #1964

**Overall change:** Integrate top story to front page section

**Code changes:**

- Added `sectionNumber` and `storyNumber` as props and use them to determined `topStory`
- Created a `StoryPromoLiContainer` to make front page section cleaner.
- Added `topStory` prop to `StoryPromo` container

---

- [x] I have assigned myself to this PR and the corresponding issues
- [ ] Tests added for new features
- [x] Test engineer approval
- [ ] I have followed [the merging checklist](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/MERGE_PROCESS.md) and this is ready to merge.

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -8350,13 +8380,13 @@
         },
         "npm-bundled": {
           "version": "1.0.6",
-          "resolved": "",
+          "resolved": false,
```
Comment (dr3) [inline package-lock.json]: Are you on the right npm? we use 6.9.0 on node 10.16.0 now

Comment (ghost) [reply package-lock.json]: Yes `10.16.0`. and npm is on version `6.9.0`

File: src/app/containers/FrontPageSection/index.jsx [hunk 2]
```diff
@@ -38,10 +55,12 @@ const FrontPageSection = ({ bar, group }) => {
...
-            </StoryPromoLi>
+          {items.map((item, index) => (
+            <StoryPromoLiContainer
+              item={item}
+              sectionNumber={sectionNumber}
+              storyNumber={index}
+            />
           ))}
         </StoryPromoUl>
       ) : (
```
Comment (DenisBBC) [inline src/app/containers/FrontPageSection/index.jsx]: If there is just one item, should it be considered as a `topStory`?

**Review by DenisHdz — APPROVED** [review]

File: src/app/containers/FrontPageSection/index.jsx (hunk 2 above)
Comment (DenisHdz) [inline src/app/containers/FrontPageSection/index.jsx:66]: If there is just one item, should it be considered as a topStory?

**Review by DenisHdz — CHANGES_REQUESTED** [review]

File: src/app/containers/FrontPageSection/index.jsx (hunk 2 above)
Comment (ghost) [reply src/app/containers/FrontPageSection/index.jsx:66]: Am not sure, better to check with product team.

File: package-lock.json (hunk 1 above)
Comment (dr3) [reply package-lock.json]: Strange, im on the same, but get different package lock
image

**Comment by andrew-nowak** [comment]
This PR is causing the bundle to balloon to 651 KiB!! #2028 will allow this to pass for now, but I'm confused what this PR is doing that causes over 40KiB of extra js to be bundled -- nothing stands out to me as an obvious cause.

Looks like those extra copies of `gel-foundations` in the package lock are a symptom --
simorgh@latest
image

simorgh@{this pr}
image

Still not sure what caused them, though

**Comment by andrew-nowak** [comment]
Resetting the package lock fixed the bundle size increase :+1:

**Review by dr3 — CHANGES_REQUESTED** [review]
This isnt working anymore, I dont think the latest story promo has been pulled in

**Review by dr3 — COMMENTED** [review]
:O This change pulls 59KiB into simorgh

**Review by dr3 — APPROVED** [review]
LGTM but we need to immediately prioritise fixing our deps problem

**Comment by j-pendlebury** [comment]
image

😱🙈😢

**Comment by ghost** [comment]
@jblaut Already raised an [issue](https://github.com/bbc/psammead/issues/668) that will make sure we use latest `gel-foundation` therefore we would not need to pull both `1.2.0` and `3.0.0`. Updating package dependacies to latest is becoming an issue now, that we have to fix.

**Review by j-pendlebury — APPROVED** [review]

**Review by DenisHdz — APPROVED** [review]

**Comment by david-boydell** [comment]
Looks good, this is ready for merge.

**Pull request merged**